Sources/HaCWebsiteLib/Events/Event.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+protocol Event {
+    var title : String {get}
+    var time : DateInterval {get}
+    var tagLine : String {get}
+    var color : String {get}
+    var hypePeriod : DateInterval {get}
+    var imageURL : String? {get}
+    var location : Location? {get} //for room names etc.
+    var facebookEventID : Double? {get}
+    var shouldShowAsUpdate : Bool {get}
```

Comment:
Perhaps we want events to have tags so we can implement a richer search later on (might be a lot of work to add tags retrospectively)